engines/sci/engine/script_patches.cpp

```diff
@@ -3322,32 +3321,36 @@ static const uint16 kq7BenchmarkSignature[] = {
 	0x38, SIG_SELECTOR16(new), // pushi new
 	0x76,                      // push0
 	0x51, SIG_ADDTOOFFSET(+1), // class Actor
-	0x4a, SIG_UINT16(0x04),    // send 4
-	0xa5, 0x00,                // sat 0
-	0x39, SIG_SELECTOR8(view), // pushi $e (view)
+	0x4a, SIG_UINT16(0x0004),  // send 4
+	0xa5, 0x00,                // sat temp[0]
+	0x39, SIG_SELECTOR8(view), // pushi view
 	SIG_MAGICDWORD,
 	0x78,                      // push1
 	0x38, SIG_UINT16(0xfdd4),  // pushi 64980
 	SIG_END
 };
 
 static const uint16 kq7BenchmarkPatch[] = {
-	0x34, PATCH_UINT16(10000), // ldi 10000
-	0x48,                      // ret
+	0x34, PATCH_UINT16(0x2710), // ldi 10000
+	0x48,                       // ret
```

Comment:
And I don't really see a point in change this either. In this case it's meant to be a specific decimal value.
Changing it to hex in the patch opcode doesn't make it easier readable, quite the opposite actually.